src/frontend/utility/footer/footer.css

```diff
@@ -0,0 +1,22 @@
+@import '../styles.css';
+
+/* Footer */
+.footer--ctr {
+  padding: 0.4rem;
+  height: min-content;
+  color: var(--primary--light);
+  background-color: var(--tertiary);
+}
+.box__image__sm{
+  border-radius: 50%;
+  padding: 0.2rem;
+  width: 2rem;
+  height: auto;
+  object-fit: contain;
+}
+.fixed {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
```

Comment:
You can use shorthand property for less lines of code

Reply:
yeah I will do that